stubs/Markdown/markdown/core.pyi

```diff
@@ -21,7 +21,7 @@ class _ReadableStream(Protocol):
 
 class Markdown:
     preprocessors: Registry[preprocessors.Preprocessor]
-    inlinePatterns: Registry[inlinepatterns.InlineProcessor]
+    inlinePatterns: Registry[inlinepatterns.Pattern]
```

Comment:
This is different to the annotations the `markdown` library now has inline:
- https://github.com/Python-Markdown/markdown/blob/5d7f2559c31844f788ea41132f40ace7555518e6/markdown/core.py#L154
- https://github.com/Python-Markdown/markdown/blob/5d7f2559c31844f788ea41132f40ace7555518e6/markdown/inlinepatterns.py#L53

Cc. @oprypin, who recently added this type hint in typeshed

Reply:
It is my mistake, actually this PR is more correct 👍

I was adding this both upstream and in typeshed btw.